### PR TITLE
Mention the concept of providers in NEWS.md and CHANGES.md

### PR DESCRIPTION
**Pull request by levitte**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -87,6 +87,7 @@ OpenSSL 3.0
   * All of the low-level DH, DSA, ECDH, ECDSA and RSA public key functions
     have been deprecated.
   * SSL 3, TLS 1.0, TLS 1.1, and DTLS 1.0 only work at security level 0.
+  * Added providers, a new pluggability concept that will replace ENGINE.
```
Comment (t8m) [inline NEWS.md:90]: Maybe `that replaces ENGINEs`?

File: CHANGES.md [hunk 2]
```diff
@@ -1389,6 +1389,14 @@ breaking changes, and mappings for the large list of deprecated functions.
 
    *Raja Ashok*
 
+ * Added a new concept for OpenSSL plugability: providers.  This
+   functionality is designed to replace ENGINE, and to be much more
```
Comment (t8m) [inline CHANGES.md:1393]: maybe `ENGINEs` or `ENGINE API`?

Comment (t8m) [reply CHANGES.md:1393]: Also I am unsure about the comma after `ENGINE`. It seems to confuse the sentence.

**Review by t8m — CHANGES_REQUESTED** [review]

File: CHANGES.md (hunk 2 above)
Comment (levitte) [reply CHANGES.md]: I went with "ENGINE API and ENGINE implementations"

**Comment by levitte** [comment]
Apart from answering the review comments, I also added a bit more text in CHANGES.md

File: CHANGES.md [hunk 3]
```diff
@@ -1389,6 +1389,23 @@ breaking changes, and mappings for the large list of deprecated functions.
 
    *Raja Ashok*
 
+ * Added a new concept for OpenSSL plugability: providers.  This
+   functionality is designed to replace the ENGINE API and ENGINE
+   implementations, and to be much more dynamic, allowing provider
```
Comment (t8m) [inline CHANGES.md:1394]: I'm still unsure about the comma before the `and`.

Comment (levitte) [reply CHANGES.md:1394]: Well, now it actually makes sense, as it shows that "ENGINE API and ENGINE implementations" are a unit.  It makes references clearer

**Review by t8m — APPROVED** [review]

**Comment by t8m** [comment]
Agree it's urgent

**Comment by levitte** [comment]
Merged

master:
8e7d941ade3a86e352d9c3d601f61c033dc6788b Mention the concept of providers in NEWS.md and CHANGES.md

3.0:
4c4ab4d7efdf8c9b49c9838742a0fcd7321d88ff Mention the concept of providers in NEWS.md and CHANGES.md

**Pull request closed without merging**